src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,52 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File getInfoFrom = new File(fromFileName);
+        StringBuilder builder = new StringBuilder();
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(getInfoFrom));
+            String value = reader.readLine();
+            while (value != null) {
+                builder.append(value).append(",");
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
```

Comment:
The `BufferedReader` is not closed after use. This can lead to resource leaks. Use try-with-resources to ensure that the reader is closed properly.